web_copy_confirm/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+=================================================
+Show a confirmation dialogue when copying records
+=================================================
```

Comment:
You can remove this bit. It is already rendered in the README.rst from the module name. In general, the readme/ fragments are not supposed to contain any RST markup at all.